srcpkgs/firefox-i18n/files/update-lang-list

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
```

Comment:
I don't think updates scripts like this should be in the void-packages repo, xtools would be more appropriate.

Reply:
Originally, I intended to add it into xtools.
However, this script is only useful for firefox and firefox-esr,
and @Chocimier told it's better to include it here.

Anyway, it's in separated commit, I'll drop it out of void-packages tree
if we can gain the consensus.

Reply:
My reason to include it here is that it is tied to one package, so
- easier to find
- xtools is useful for non-maintainers - we ask for xuname output for bug reports - and populating /bin with package-tied scripts for them isn't great
- we have similar scripts for bash, iana-etc.

Reply:
I agree with @Chocimier , having such scripts in the tree is highly useful. I've used the glibc update script in the past to update upstream patches which is also in tree.